### PR TITLE
LPS-140315 It is not possible to edit the header image alt text on a blog post

### PR DESCRIPTION
**Pull request by boton**
Bug: https://issues.liferay.com/browse/LPS-140315

This PR adds a description to the cover background image on a blog entry using the cover image caption. 

We can improve it in the future by using the DM file description (adding Documents 
 & Media file description to blogs images) or adding a field in the blog entry.

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/asset/full_content.jsp [hunk 1]
```diff
@@ -40,7 +40,12 @@ String entryTitle = BlogsEntryUtil.getDisplayTitle(resourceBundle, entry);
 				%>
 
 				<c:if test="<%= Validator.isNotNull(coverImageURL) %>">
-					<div aria-label="<%= HtmlUtil.escapeAttribute(HtmlUtil.stripHtml(entry.getCoverImageCaption())) %>" class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" role="img" style="background-image: url(<%= coverImageURL %>);"></div>
+
+					<%
+					String coverImageCaption = HtmlUtil.escapeAttribute(HtmlUtil.stripHtml(entry.getCoverImageCaption()));
+					%>
+
+					<div <c:if test="<%= Validator.isNotNull(coverImageCaption) %>"> aria-label="<%= coverImageCaption %>" role="img" </c:if> class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" style="background-image: url(<%= coverImageURL %>);"></div>
```
Comment (AliciaGarciaGarcia) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/asset/full_content.jsp:48]: I think that I see on jsps more the ternary operator 
`<div <%= Validator.isNotNull(coverImageCaption) ? "aria-label=\"" + coverImageCaption + "role=\"img\"" : StringPool.BLANK%> class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" style="background-image: url(<%= coverImageURL %>);"></div>`

than with this way

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia @boton Is there anything else to do here? can this be forwarded?

**Comment by boton** [comment]
I’m not sure if is good enough to forward it because we only adding redundant information to screen readers ¯\_(ツ)_/¯

**Comment by adolfopa** [comment]
Yes, I see no problem forwarding this.

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**